irohad/network/impl/block_loader_impl.cpp

```diff
@@ -96,7 +104,7 @@ nonstd::optional<Block> BlockLoaderImpl::retrieveBlock(
   protocol::Block block;
 
   // request block with specified hash
-  request.set_hash(block_hash.to_string());
+  request.set_hash(toBinaryString(block_hash));
```

Comment:
What is the reason to change hashing here?

Reply:
`to_string` in SM returns human-readable string, not binary